config.go

```diff
@@ -547,7 +546,7 @@ func readGroupConfiguration() error {
 		// verify members also exist within config
 		for i = range currentGroup.Members {
 			match = 0
-			for dn = range localPeople {
+			for dn := range localPeople {
```

Comment:
Please stick with the existing style. While it is a perfectly valid line, I'd like to see all variables be defined at the beginning of a function to help keep an overview of what exists and add notes to the use of a var (although not really done yet).

Reply:
sure, no problem with that